metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -628,5 +639,33 @@ public void apply(final Metafix metafix, final Record record, final List<String>
     private static final String ERROR_STRING_OPTION = "error_string";
 
     private static final Random RANDOM = new Random();
+    private static String defaultValue;
+
+    private static Map<String, String> extracted(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options, final String kindOfMap) {
+        final Map<String, String> map;
+        if (params.size() <= 1) {
+            map = options;
```

Comment:
I'm not sure if every lookup should support local maps. This should probably be limited to `lookup`. Maybe through an additional parameter `boolean optionsAsMap`.

Reply:
don't know. Maybe you could implement that? I see no harm here as it is.

Reply:
Yep, right. Implement please.

Reply:
Obsolete with dropping special treatment of RDF maps in `lookup()` (0987eb0).